########## Testcase 0: Failed ##########
Expected Output: 
Yes
Your Output: 
true
false
false
false
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
########## Testcase 1: Failed ##########
Expected Output: 
No
Your Output: 
false
false
false
false
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
########## Testcase 2: Failed ##########
Expected Output: 
No
Your Output: 
false
false
false
false
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
########## Testcase 3: Failed ##########
Expected Output: 
No
Your Output: 
false
false
false
false
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
########## Testcase 4: Failed ##########
Expected Output: 
Yes
Your Output: 
false
false
false
false
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
########## Testcase 5: Failed ##########
Expected Output: 
Yes
Your Output: 
true
true
true
true
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 4
	at HashTableChain.compare(Solutions.java:50)
	at Solutions.main(Solutions.java:169)
----------------------------------------
